torch_geometric/loader/link_neighbor_loader.py

```diff
@@ -258,7 +305,12 @@ def __init__(
         edge_type, edge_label_index = get_edge_label_index(
```

Comment:
I think this might break if your graph has no edges?

Reply:
I do have tests for graphs without edges, it works fine for them. Do you have some specific case in mind

Reply:
sorry I meant if `edge_label_index = None` (default)

Reply:
Good point, I think we should force the user the pass an `edge_label_index` if the graph doesn't have edges. Maybe with an assert statement.